content/_index.md

```diff
@@ -73,40 +69,42 @@ g3Commons:
     link: https://www.bloodpac.org/
   c3:
     fig: figs/commons/pdc.png
-    link: /
-  c4:
-    fig: figs/commons/braincommons.png
-    link: https://www.braincommons.org/
+    link: https://proteomic.datacommons.cancer.gov/pdc/
 g3Intro:
   title: Introduction to Gen3
-  detail: Gen3 is an open source software with Apache 2.0 or similar licenses, colocating compute and storage in a data commons. It is agnostic to the data type and the storage location, requiring  minimally, a data model, data, a secure landing page for the portal, and research goals.
+  detail: The Gen3 Data Platform is an open-source (with an Apache License) data platform for developing and operating cloud-based data commons, workspaces, and an ecosystem of interconnected, cloud-based data resources, applications and notebooks. A functioning data ecosystem empowers researchers in the same field to share and combine datasets for reanalysis and reuse as new tools and hypotheses are developed.  The Gen3 Data Platform consists of Gen3 Framework Services, Gen3 Data Commons and Gen3 Workspaces.
   fig: figs/productkv.svg
```

Comment:
Saw that you used the Oxford Comma above so: 
"applications, and notebooks".
"Gen3 Data Commons, and Gen3 Workspaces."

I might also suggest to change the first sentence into:
"...an open-source data platform with Apache License..."

And, really minor: two spaces before the beginning of the last sentence